Makefile

```
@@ -0,0 +1,52 @@
+ARCHS_IOS = x86_64-apple-ios aarch64-apple-ios aarch64-apple-ios-sim
+ARCHS_ANDROID = i686-linux-android x86_64-linux-android aarch64-linux-android armv7-linux-androideabi
+LIB = libwcgcore.dylib
+XCFRAMEWORK = WCGCore.xcframework
+RUST_LIB = wcgcore
+
+all:GENERATE_HEADERS ios android
+
+ios: $(XCFRAMEWORK)
+
+android: GENERATE_ANDROID
+
+
+.PHONY: android_only
+android_only: android
+
+.PHONY: ios_only
+ios_only: GENERATE_HEADERS ios BUILD_XCFRAMEWORKS
+
+.PHONY: GENERATE_HEADERS
+GENERATE_HEADERS:
+	./tools/scripts/build-rust-headers.sh
+
+# PHONY keyword on make means this is not a file, just an identifier for a target
+.PHONY: $(ARCHS_IOS)
+$(ARCHS_IOS): %:
+	cargo +nightly build -Z build-std='std,panic_abort'  -Z build-std-features=panic_immediate_abort --target $@ --release -p ios
+
+$(XCFRAMEWORK): $(ARCHS_IOS)
+	rm -rf target/simulator_fat
+	mkdir target/simulator_fat
```

In case someone decides to manually delete the `target` folder, don't fail on it not existing.
```suggestion
	mkdir -vp target/simulator_fat
```

---

Nice, but the target should exist due to previous build cmd of `ARCHS_IOS `